.env.sample

```
@@ -657,6 +657,8 @@ MAX_STREAM_EXPRESSIONS_TOTAL_CHARACTERS=50000
 MAX_SEL_LENGTH=3000
 # The maximum character length for formatter templates.
 MAX_FORMATTER_TEMPLATE_LENGTH=5000
+# Maximum number of streams that can be precached at once when precacheSingleStream is disabled.
+MAX_PRECACHE_STREAMS=10
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MAX_PRECACHE_STREAMS key should go before the MAX_SEL_LENGTH key

(UnorderedKey)